src/components/FormProvider/FormProvider.jsx

```diff
@@ -1,13 +1,9 @@
-import React, { createContext, useContext, useState } from 'react';
+import React from 'react';
+import { MuiPickersUtilsProvider } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
 
-const FormContext = createContext([]);
-
-const FormProvider = ({ children, value }) => {
-  const [form, setForm] = useState(value ?? {});
-  return <FormContext.Provider value={{ form, setForm }}>{children}</FormContext.Provider>;
-};
-
-const useForm = () => useContext(FormContext);
+function FormProvider({ children }) {
```

Comment:
Is this really still a form provider?  It seems like its more a date utils provider if anything.

I could see it still being a FormProvider if we moved `<Formik />` and maybe `<Form />` here, which I think would make sense.  That could also help reduce our coupling to this (or any other) implementation.  If we do this some of the removed tests could still apply, even if they'll be testing the library more than anything else (e.g. tests around intial values or state updates).

Reply:
If we wanted to go crazy on that front setting we could also set`useForm = useFormikContext` to contain the Formik dependency, but I don't think that's necessary at this point.

Reply:
I thought about moving the date utils provider to the app and just getting rid of this file, but it's used so much in the tests that it seemed simpler to leave it be. I do think it makes sense to move the Formik-related providers here as well, so I'll do that.